consensus/src/simplex/actors/batcher/actor.rs

```diff
@@ -316,7 +42,7 @@ pub struct Actor<
     reporter: R,
 
     activity_timeout: ViewDelta,
-    skip_timeout: ViewDelta,
+    skip_timeout: usize,
```

Comment:
I guess this is done to avoid the `.get()` later on, but instead we introduce a `usize::from()`, not sure that's worth it.